components/dataproducts/dataset/datasetMetadata.tsx

```diff
@@ -22,6 +24,7 @@ const DatasetMetadata = ({ datasource }: DataproductTableSchemaProps) => {
     { k: 'Tabell', v: datasource.table },
     { k: 'Tabelltype', v: datasource.tableType.toUpperCase() },
     { k: 'Opprettet', v: humanizeDate(datasource.created) },
+    { k: 'Link til kildekode', v: dataset.repo || 'Ukjent' },
```

Comment:
Should it be a link with external link icon, and not just a string?

Reply:
i agree with erik, e.g.

```suggestion
    { k: 'Link til kildekode', v: dataset.repo ? <Link target="_blank" rel="norefferer" href={dataset.repo}>{dataset.repo} <ExternalLink /></Link> : '' },
```

Reply:
then later on line ~46, i think it's nicer if we just don't show the section if it's not defined:

```typescript
        {entries.filter(({v}) => v).map(({ k, v, copy }, idx) => (
```

you can choose to make the filter more explicit if you want, e.g. `.filter(({v}) => v !== '')`

Reply:
Cool, I accepted your change, and just chnaged a little with the filtering